indico/modules/rb/client/js/common/rooms/edit/RoomEditOptions.jsx

```diff
@@ -0,0 +1,162 @@
+            name="available_equipment"
+            component={EquipmentList}
+            isEqual={_.isEqual}
+            componentLabel={Translate.string('Add new equipment')}
+            hideErrorPopup={!active}
+          />
+        )}
+        <FieldArray name="attributes" isEqual={_.isEqual}>
+          {({fields}) => {
+            if (!fields.value) {
+              return null;
+            }
+            const fieldsByName = fields.map(x => x.name);
+            const options = globalAttributes
+              .map(x => ({
+                key: x.name,
+                text: x.title,
+                value: x.name,
+              }))
+              .filter(attr => !fieldsByName.includes(attr.key));
+            return (
+              <div>
+                <Dropdown
+                  className="icon room-edit-modal-add-btn"
+                  button
+                  text={Translate.string('Add new attributes')}
+                  floating
+                  labeled
+                  icon="add"
+                  options={options}
+                  search
+                  disabled={options.length === 0}
+                  selectOnBlur={false}
+                  value={null}
+                  onChange={(__, {value: name}) => {
+                    fields.push({value: null, name});
+                  }}
+                />
+                {fields.map((attribute, index) => (
+                  <FinalInput
+                    key={attribute}
+                    name={`${attribute}.value`}
+                    label={attributeTitles[fields.value[index].name]}
+                    hideErrorPopup={!active}
+                    required
+                    icon={{
+                      name: 'remove',
+                      color: 'red',
+                      link: true,
+                      onClick: () => fields.remove(index),
+                    }}
+                  />
+                ))}
+                {fields.length === 0 && (
+                  <div>
+                    <Translate>No custom attributes found</Translate>
+                  </div>
+                )}
+              </div>
+            );
+          }}
+        </FieldArray>
+      </Form.Group>
+      <Header>
+        <Translate>Options</Translate>
+      </Header>
+      <Form.Group grouped>
+        <FinalCheckbox name="is_reservable" label={Translate.string('Bookable')} hideErrorPopup />
+        <FinalCheckbox
+          name="reservations_need_confirmation"
+          label={Translate.string('Require confirmation (pre-bookings)')}
+          hideErrorPopup={!active}
+        />
+        <FinalCheckbox
+          name="notifications_enabled"
+          label={Translate.string('Reminders enabled')}
+          hideErrorPopup={!active}
+        />
+        <FinalCheckbox
+          name="end_notifications_enabled"
+          label={Translate.string('Reminders of finishing bookings enabled')}
+          hideErrorPopup={!active}
+        />
```

Comment:
Should those be here or on the notifications tab? I'd be inclined to put the actual toggleable checkboxes on the notifications tab, and show some kind of placeholder (disabled checkbox?) here indicating the current state and maybe a link that switches to the notifications tab where one can change them.

Right now it's a bit confusing to not see some fields in the notifications tab based on a checkbox in a different tab - you have no way of knowing that toggling them will show/enable fields elsewhere.

Reply:
Yes, it is confusing from a user point of view. Imo, I think this should be in the notifications tab without any links in the other tab. I don't think the assumption of considering it a (generic) option was correct anyway.